### ISSUE TITLE
欢迎分享CVPR 2025 论文和代码 / Welcome to share the paper and code of CVPR 2025

### ISSUE THREAD

**Issue by amusi** [amusi/CVPR2025-Papers-with-Code#242]
[The format of the issue]
Paper title:
Paper link:
Code link:

**Comment by amusi** [comment]
[Sample]
MambaVision: A Hybrid Mamba-Transformer Vision Backbone
Paper: https://arxiv.org/abs/2407.08083
Code: https://github.com/NVlabs/MambaVision

**Comment by Xiangxu-0103** [comment]
LiMoE: Mixture of LiDAR Representation Learners from Automotive Scenes
Paper: https://arxiv.org/abs/2501.04004
Code: https://github.com/Xiangxu-0103/LiMoE
Project: https://ldkong.com/LiMoE

**Comment by Epiphqny** [comment]
PAR: Parallelized Autoregressive Visual Generation
Paper: https://arxiv.org/abs/2412.15119
Code: https://github.com/Epiphqny/PAR
Project: https://epiphqny.github.io/PAR-project/

**Comment by MingkunLei** [comment]
StyleStudio: Text-Driven Style Transfer with Selective Control of Style Elements
Paper: https://arxiv.org/abs/2412.08503
Code: https://github.com/Westlake-AGI-Lab/StyleStudio
Project: https://stylestudio-official.github.io/

**Comment by 2toinf** [comment]
Universal Actions for Enhanced Embodied Foundation Models
Paper: https://arxiv.org/abs/2501.10105
Code: https://github.com/2toinf/UniAct
Project: https://2toinf.github.io/UniAct/

**Comment by Fediory** [comment]
HVI: A New color space for Low-light Image Enhancement
Paper: https://arxiv.org/abs/2502.20272
Code: https://github.com/Fediory/HVI-CIDNet
Demo: https://huggingface.co/spaces/Fediory/HVI-CIDNet_Low-light-Image-Enhancement_

**Comment by LiewFeng** [comment]
Paper title: Timestep Embedding Tells: It's Time to Cache for Video Diffusion Model
Paper link: https://arxiv.org/abs/2411.19108
Code link: https://github.com/ali-vilab/TeaCache
Project: https://liewfeng.github.io/TeaCache/
Topic: Visual Generation Acceleration

**Comment by wbhu** [comment]
DepthCrafter: Generating Consistent Long Depth Sequences for Open-world Videos
Paper: https://arxiv.org/abs/2409.02095
Code: https://github.com/Tencent/DepthCrafter
Project: https://depthcrafter.github.io
Topic: 深度估计(Depth Estimation)

**Comment by hzxie** [comment]
Generative Gaussian Splatting for Unbounded 3D City Generation

Paper: https://arxiv.org/abs/2406.06526
Code: https://github.com/hzxie/GaussianCity
Project: https://haozhexie.com/project/gaussian-city
Topic: [https://github.com/amusi/CVPR2025-Papers-with-Code#3D-Generation](3D生成), [3DGS (Gaussian Splatting)](https://github.com/amusi/CVPR2025-Papers-with-Code#3DGS)

**Comment by callsys** [comment]
Paper title: DynRefer: Delving into Region-level Multimodal Tasks via Dynamic Resolution
Paper link: https://arxiv.org/abs/2405.16071
Code link: https://github.com/callsys/DynRefer
Topic: Multimodal learning, MLLM

**Comment by MqLeet** [comment]
ReDDiT: Efficient Diffusion as Low Light Enhancer
Paper: https://arxiv.org/abs/2410.12346
Code: https://github.com/lgz-0713/ReDDiT

**Comment by yuanc3** [comment]
From Poses to Identity: Training-Free Person Re-Identification via Feature Centralization
Paper: https://arxiv.org/abs/2503.00938
Code: https://github.com/yuanc3/Pose2ID
Topic: ReID

**Comment by pandayuanyu** [comment]
Generative Photography: Scene-Consistent Camera Control for Realistic Text-to-Image Synthesis

Paper: https://arxiv.org/abs/2412.02168
Code: https://github.com/pandayuanyu/generative-photography
Project Page: https://generative-photography.github.io/project/
Dataset: https://huggingface.co/datasets/pandaphd/camera_settings
Demo: https://huggingface.co/spaces/pandaphd/generative_photography

Topic: Image / Video Generation, Camera Physics

**Comment by hyz317** [comment]
StdGEN: Semantic-Decomposed 3D Character Generation from Single Images
Paper: https://arxiv.org/abs/2411.05738
Code: https://github.com/hyz317/StdGEN
Project Page: https://stdgen.github.io/
Huggingface: https://huggingface.co/hyz317/StdGEN
Topic: 3D Generation, Avatar

**Comment by csuhan** [comment]
Retrieval-Augmented Personalization for Multimodal Large Language Models
Paper: https://arxiv.org/abs/2410.13360
Code: https://github.com/Hoar012/RAP-MLLM
Project Page: https://hoar012.github.io/RAP-Project/
Topic: MLLM

**Comment by nktoan** [comment]
"_h_-Edit: Effective and Flexible Diffusion-Based Editing via Doob’s _h_-Transform"
Paper: https://arxiv.org/abs/2503.02187
Code: https://github.com/nktoan/h-edit
Topic: Image Editing/ Manipulation/ Generation

**Comment by qunovo** [comment]
NLPrompt: Noise-Label Prompt Learning for Vision-Language Models
Paper: https://arxiv.org/abs/2412.01256
Code: https://github.com/qunovo/NLPrompt
Topic: Vision-Language Models/ Prompt Learning/ Nosie Label

**Comment by xifen523** [comment]
[The format of the issue]
Paper title:Omnidirectional Multi-Object Tracking
Paper link:https://arxiv.org/abs/2503.04565
Code link:https://github.com/xifen523/OmniTrack

**Comment by xifen523** [comment]
Paper title:Omnidirectional Multi-Object Tracking
Paper link:https://arxiv.org/abs/2503.04565
Code link:https://github.com/xifen523/OmniTrack
Topic: Object Tracking

**Comment by yongliang-wu** [comment]
Number it: Temporal Grounding Videos like Flipping Manga
Paper: https://arxiv.org/abs/2411.10332
Code: https://github.com/yongliang-wu/NumPro
Topic: Vision-Language Models/ Video Understanding/ MLLM

**Comment by LanceZPF** [comment]
OpenING: A Comprehensive Benchmark for Judging Open-ended Interleaved Image-Text Generation
Paper: https://arxiv.org/abs/2411.18499).
Code: https://github.com/LanceZPF/OpenING
Project Page: https://opening-benchmark.github.io/
Topic: Datasets and Benchmarks

**Comment by TahaKoleilat** [comment]
BiomedCoOp: Learning to Prompt for Biomedical Vision-Language Models
Paper: https://arxiv.org/abs/2411.15232
Code: https://github.com/HealthX-Lab/BiomedCoOp
Topic: Vision-Language Models/ Medical Imaging / Prompt Learning

**Comment by unira-zwj** [comment]
PhysVLM: Enabling Visual Language Models to Understand Robotic Physical Reachability
Paper link: https://arxiv.org/abs/2503.08481
Code link: https://github.com/unira-zwj/PhysVLM
Topic: 具身智能(Embodied AI)/ 多模态大语言模型(MLLM)/ 

您的首页README有一个Embodied AI和一个具身智能(Embodied AI)的list，好像重复了

**Comment by FengodChen** [comment]
Using Powerful Prior Knowledge of Diffusion Model in Deep Unfolding Networks for Image Compressive Sensing
Paper: https://arxiv.org/abs/2503.08429
Code: https://github.com/FengodChen/DMP-DUN-CVPR2025
Topic: Diffusion Models / Low-level Vision

**Comment by 924973292** [comment]
IDEA: Inverted Text with Cooperative Deformable Aggregation for Multi-modal Object Re-Identification
Paper: https://arxiv.org/abs/2503.10324
Code: https://github.com/924973292/IDEA
Topic: Multi-modal Fusion / MLLM / Object ReID

**Comment by unira-zwj** [comment]
PhysVLM: Enabling Visual Language Models to Understand Robotic Physical Reachability
Paper: https://arxiv.org/abs/2503.08481
Code: https://github.com/unira-zwj/PhysVLM
Topic: Embodied AI/ MLLM

**Comment by peakpang** [comment]
Unlocking Generalization Power in LiDAR Point Cloud Registration
Paper: https://arxiv.org/abs/2503.10149
Code: https://github.com/peakpang/UGP
Topic: 3D Registration

**Comment by yunncheng** [comment]
MMRL: Multi-Modal Representation Learning for Vision-Language Models
Paper: https://arxiv.org/abs/2503.08497
Code: https://github.com/yunncheng/MMRL
Topic: CLIP

**Comment by qianlong0502** [comment]
Paper title: DKDM: Data-Free Knowledge Distillation for Diffusion Models with Any Architecture
Paper link: https://arxiv.org/abs/2409.03550
Code link: https://github.com/qianlong0502/DKDM

**Comment by jinli99** [comment]
Paper title: DTGBrepGen: A Novel B-rep Generative Model through Decoupling Topology and Geometry
Paper link: https://arxiv.org/abs/2503.13110
Code link: https://github.com/jinli99/DTGBrepGen
Topic: CAD Generation

**Comment by youkangGG** [comment]
Paper title: RENO: Real-Time Neural Compression for 3D LiDAR Point Clouds
Paper link: https://arxiv.org/abs/2503.12382
Code link: https://github.com/NJUVISION/RENO
Topic: 3D点云(3D Point Cloud)

**Comment by feufhd** [comment]
Paper title: Building Vision Models upon Heat Conduction
Paper link: https://arxiv.org/abs/2405.16555
Code link: https://github.com/MzeroMiko/vHeat
Topic: Visual Representation

**Comment by AuroraZengfh** [comment]
Paper title: MambaIC: State Space Models for High-Performance Learned Image Compression
Paper link: https://arxiv.org/abs/2503.12461
Code link: https://arxiv.org/abs/2503.12461
Topic: image compression, vision, mamba

**Comment by Yore0** [comment]
Test-Time Domain Generalization via Universe Learning: A Multi-Graph Matching Approach for Medical Image Segmentation
Paper: https://arxiv.org/abs/2503.13012
Code: https://github.com/Yore0/TTDG-MGM
Topic: Medical Image Segmentation, Test-time Adaptation

**Comment by delaprada** [comment]
Analyzing the Synthetic-to-Real Domain Gap in 3D Hand Pose Estimation
Paper: https://arxiv.org/abs/2503.19307
Code: https://github.com/delaprada/HandSynthesis.git
Topic: 3D Hand Pose Estimation / Synthetic Data Generation

**Comment by shaohao011** [comment]
BrainMVP: Multi-modal Vision Pre-training for Medical Image Analysis
Paper: https://arxiv.org/abs/2410.10604
Code: https://github.com/shaohao011/BrainMVP

**Comment by jameslahm** [comment]
LSNet: See Large, Focus Small
Paper: https://arxiv.org/abs/2503.23135
Code: https://github.com/jameslahm/lsnet
Topic: Backbone

**Comment by shaohao011** [comment]
BrainMVP: Multi-modal Vision Pre-training for Medical Image Analysis
Paper: https://arxiv.org/abs/2410.10604
Code: https://github.com/shaohao011/BrainMVP
Topic: Medical Image Analysis, Multi-modal representation learning, missing modality

**Comment by zhangchbin** [comment]
Mr. DETR: Instructive Multi-Route Training for Detection Transformers
Paper: https://arxiv.org/abs/2412.10028
Code: https://github.com/Visual-AI/Mr.DETR

**Comment by ADu2021** [comment]
AdaMMS: Model Merging for Heterogeneous Multimodal Large Language Models with Unsupervised Coefficient Optimization
Paper: https://arxiv.org/abs/2503.23733
Code: https://github.com/THUNLP-MT/AdaMMS
Topic: MLLM

**Comment by weixiang-zhang** [comment]
EVOS: Efficient Implicit Neural Training via EVOlutionary Selector
Paper: https://arxiv.org/abs/2412.10153
Code: https://github.com/zwx-open/EVOS-INR
Project Page: https://weixiang-zhang.github.io/proj-evos/
Topic: INR

**Comment by madaror** [comment]
Tiled Diffusion
Paper link: https://arxiv.org/abs/2412.15185
Code link: https://github.com/madaror/tiled-diffusion
Project Page: https://madaror.github.io/tiled-diffusion.github.io/
Topic: Image Generation

**Comment by HalleyJiang** [comment]
Title: DEFOM-Stereo: Depth Foundation Model Based Stereo Matching
Paper: https://arxiv.org/abs/2501.09466
Code: https://github.com/Insta360-Research-Team/DEFOM-Stereo
Project Page: https://insta360-research-team.github.io/DEFOM-Stereo/
Topic: Stereo Matching; Depth Estimation

**Comment by Kamichanw** [comment]
Mimic In-Context Learning for Multimodal Tasks
Paper: https://arxiv.org/abs/2504.08851
Code: https://github.com/Kamichanw/MimIC/
Topic: Vision Language Model; In-Context Learning

**Comment by theodore-zhao** [comment]
Boltzmann Attention Sampling for Image Analysis with Small Objects
Paper: https://arxiv.org/abs/2503.02841
Code: https://github.com/microsoft/BoltzFormer
Topic: Semantic Segmentation, Medical Image, Medical Image Segmentation

**Comment by ChendiDotLin** [comment]
Add Objaverse++ dataset 
title: Objaverse++: Curated 3D Object Dataset with Quality Annotations
paper: https://arxiv.org/abs/2504.07334
code: https://github.com/TCXX/ObjaversePlusPlus

**Comment by Suo-Wei** [comment]
title: Octopus: Alleviating Hallucination via Dynamic Contrastive Decoding (highlight)
paper: https://arxiv.org/abs/2503.00361
code: https://github.com/LijunZhang01/Octopus

**Comment by HailinYu0414** [comment]
Title: From Sparse to Dense: Camera Relocalization with Scene-Specific Detector from Feature Gaussian Splatting
Paper: https://arxiv.org/abs/2503.19358
Code: https://github.com/zju3dv/STDLoc
Project Page: https://zju3dv.github.io/STDLoc/
Topic: Visual Localization, 3DGS

**Comment by heekhero** [comment]
Title: Quantization without Tears
Paper: https://arxiv.org/abs/2411.13918
Code: https://github.com/wujx2001/QwT
Topic: Model Pruning and Quantization

**Comment by ty625911724** [comment]
Title: SketchVideo: Sketch-based Video Generation and Editing
Paper: https://arxiv.org/abs/2503.23284
Code: https://github.com/IGLICT/SketchVideo
Topic: Video Generation/ Video Editing

**Comment by yuliangguo** [comment]
Title: Depth Any Camera: Zero-Shot Metric Depth Estimation from Any Camera
Paper: https://arxiv.org/abs/2501.02464
Code: https://github.com/yuliangguo/depth_any_camera
Topic: Depth Estimation
